core/models.py

```diff
@@ -167,6 +167,15 @@ def __unicode__(self):
     class Meta:
         ordering = ('position', )
 
+    def logo_display_for_admin(self):
+        if self.logo:
+            return '<a href="{}" target="_blank"><img src="{}" alt="{}" width="100" /></a>'.format(
```

Comment:
Doesn't that create escaping issues? I would use `django.utils.html.format_html` instead of `str.format`.